src/bundle/Resources/public/scss/_extra-actions.scss

```diff
@@ -121,7 +121,8 @@
 
     &--create {
         #{$block}__content {
-            max-height: initial;
+            max-height: 90%;
```

Comment:
Why "90%"? There might be some resolution where 90% will be more than the available space. I would say it should be `calc(100% - Xpx)`.